### DISCUSSION TITLE
What does "__p9k_trapint" do?

### DISCUSSION THREAD

**Discussion by Kritiqual** [romkatv/powerlevel10k#2779] [General]
When ever I open a new terminal, the exit code indicator always show an error, after figuring that p9k_trapint keep throwing exit code 130. I wonder what does this do? Is it safe to remove it? If yes, is there any way to disable it? If no, is there any workaround?

**Comment by romkatv** [comment]
> When ever I open a new terminal, the exit code indicator always show an error

You can either figure out why the last command in your startup files fails, or mask the error by adding `true` at the bottom of `.zshrc`.
